models/AppState.py

```diff
@@ -14,10 +86,70 @@ def __init__(self):
         self.last_buy_size = 0
         self.last_buy_price = 0
         self.last_buy_filled = 0
-        #self.last_buy_value = 0
         self.last_buy_fee = 0
         self.last_buy_high = 0 
         self.last_df_index = ''
         self.sell_count = 0
         self.sell_sum = 0
         self.first_buy_size = 0
+
+    @classmethod
+    def initLastAction(self, app:PyCryptoBot, account:TradingAccount, state):
```

Comment:
It would be better if we use cls as a first parameter for classmethod intead of self.

https://stackoverflow.com/questions/4613000/difference-between-cls-and-self-in-python-classes

Reply:
+1 to respect standards

Reply:
Nice, thanks. I wasn't aware of the difference but definitely want to follow standards.